src/Network/Network/Cortex/VirtualHubRouteTable/AddAzureRmVirtualHubRouteCommand.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.Azure.Commands.Network
     [Cmdlet(
         VerbsCommon.Add,
         ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VirtualHubRoute",
-        SupportsShouldProcess = false),
+        SupportsShouldProcess = true),
```

Comment:
Since this cmdlet creates in-memory object and doesn't send any requests, it shouldn't support ShouldProcess. Please add this to statis analysis exceptions

Reply:
done